cmd/storage-rpc-client.go

```diff
@@ -356,15 +369,17 @@ func (n *networkStorage) ReadAll(volume, path string) (buf []byte, err error) {
 		return nil, errFaultyRemoteDisk
 	}
 
-	if err = n.rpcClient.Call("Storage.ReadAllHandler", &ReadAllArgs{
-		Vol:  volume,
-		Path: path,
-	}, &buf); err != nil {
+	args := ReadAllArgs{Vol: volume, Path: path}
```

Comment:
We could have called `Storage.ReadAllHandler` from the `verifyServerFunc()` directly. That way we can avoid incrementing `networkIOErrCount`